server/src/db/migrations/20210201204619-create-user.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
```

Comment:
if you put a default value, sequelize will generate a uuid on create. Otherwise you will have to manually generate it

Reply:
set a default value